components/modules/QuestionForm.js

```diff
@@ -277,6 +279,12 @@ const QuestionForm = (props) => {
       <Box p={4}>
```

Comment:
might want to add a top margin to this button. Spacing looks weird with addition text underneath